web/ui/src/common/miscUtils.js

```diff
@@ -207,6 +207,14 @@
                 return childCount;
             },
 
+            validateHostName: function(hostStr){
```

Comment:
Actually, we need to add another validation. For Port, the only valid values are positive, non-zero, unsigned 16-bit ints